cli/azd/pkg/tools/azcli/spring_app.go

```diff
@@ -0,0 +1,49 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/appplatform/armappplatform"
+)
+
+type AzCliSpringAppProperties struct {
+	HostNames []string
+}
+
+func (cli *azCli) GetSpringAppProperties(
+	ctx context.Context,
+	subscriptionId, resourceGroup, appName string,
+) (*AzCliSpringAppProperties, error) {
+	client, err := cli.createSpringAppClient(ctx, subscriptionId)
+	if err != nil {
+		return nil, err
+	}
+
+	// use "default" app name
+	springApp, err := client.Get(ctx, resourceGroup, appName, "default", nil)
```

Comment:
Let's make "default" a parameter of this method.  In doing so, let's rename the existing `appName` property to `serviceName` (so we are closer to the SDK) and then move the policy of passing "default" into `service_target_springapp`.